src/content/docs/new-relic-control/agent-control/upgrading.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: "Upgrading Agent Control and Managed Agents"
+metaDescription: "A simple, centralized process for updating agent versions across your entire fleet"
+freshnessValidatedDate: never
+---
+
+<Callout title="preview">
+  We're still working on this feature, but we'd love for you to try it out!
+
+  This feature is currently provided as part of a preview program pursuant to our [pre-release policies](/docs/licenses/license-information/referenced-policies/new-relic-pre-release-policy).
+</Callout>
+
+## Overview
+
+Agent Control provides a seamless and centralized way to manage the entire lifecycle of your agents, including updates. Instead of manually updating agents on each host or cluster, you define your desired state in New Relic, and Agent Control automatically handles the rest.
+
+The update process for both Agent Control and the agents it manages is designed to be non-disruptive and automated.
+
+## Updating Your Managed Agents
+You can update any agent managed by Agent Control by simply changing its version in the remote configuration within New Relic Control. This declarative approach ensures consistency and reliability across your entire fleet.
+
+- **Edit the Configuration:** Navigate to the configuration for your fleet in New Relic Control.
+- **Update the Version:** Locate the agent you want to update (e.g., `infrastructure`)` and change the `chart_version` value to the new version number.
+- **Deploy the Changes:** Save and deploy the updated configuration.
+
+Agent Control will automatically detect the change and perform a non-disruptive, rolling update on your cluster, ensuring that your new agents are running and healthy before replacing the old ones.
+
+### Example: Upgrading the New Relic Infrastructure Agent
+
+This example shows how to upgrade the Infrastructure agent to a new version.
+
+```shell
+# Remote configuration for a single agent
+chart_version: "5.0.109"  # Change this value
+chart_values:
+  newrelic-infrastructure:
+    enableProcessMetrics: true
+```
+
+## Updating Agent Control
+Agent Control is capable of updating itself remotely. This ensures that the supervisor running on your cluster is always up to date with the latest features, performance improvements, and security patches.
+
+The process for updating the Agent Control supervisor is the same as for any other managed agent: you specify the new version in the configuration and deploy the changes from New Relic Control.
+
+1. Edit the Agent Control Configuration: In New Relic Control, access the core configuration for Agent Control itself.
+2. Update the `chart_version`: Change the `chart_version` to the new desired version.
+3. Deploy the Changes: Save and deploy the configuration.
+
+This process triggers a safe, rolling update of the Agent Control pods on your cluster. Kubernetes' built-in health checks ensure the new version is healthy before the old one is terminated, guaranteeing a non-disruptive upgrade.
```

Comment:
We'll need to review this when the UI is ready, AFAIK this cannot be done in a deployment for now.